shared/css/typography.scss

```diff
@@ -1,5 +1,9 @@
 @import "color", "font", "barlow-semi-condensed-font";
 
+/* This file is a adopted copy of apps/src/componentLibrary/typography/typography.module.scss
```

Comment:
Usually only one of the duplicates will be the adopted one :)

Reply:
Yeah, that's the tricky part here. Those two files are linked one to each other, so that new version/update can be done on any of them first. Meaning if you update one - you should at least update version field so that we can compare 2 files and see which one is the newest.

Agree that it might be confusing, wondering how to fix that. Is something like `This file is linked copy of apps/src/... ` looks better to you?

Reply:
Updated` typography.module.scss ` comment to```
 /* This file is being copied and adopted to shared/css/typography.scss
   Updating styles in this file will require a manual sync of the that file.
*/
```